code/src/components/UserInput.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState, useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { game } from "reducers/game";
```

Comment:
This could be moved down to the other local imports, and don't forget to add './' to indicate path.